app/services/notify_service.rb

```diff
@@ -34,7 +107,16 @@ def dismiss?
       blocked ||= @recipient.muting_notifications?(@sender)
       blocked ||= conversation_muted?
       blocked ||= blocked_mention? if message?
-      blocked
+
+      return true if blocked
+      return false unless Notification::PROPERTIES[@notification.type][:filterable]
```

Comment:
Potential nil error here if there's ever a notification type we forget to add to the properties hash there, maybe an assertion in the initializer could make sense?

Reply:
The check is not new, it was already in `FilterCondition` (pushed a commit to refactor that), but I understand the concern. I do not understand your suggestion, though

Reply:
I was thinking something like `raise ArgumentError unless Notification::PROPERTIES.key?(@notification.type)`